src/main/java/wrm/AbstractSassMojo.java

```diff
@@ -1,27 +1,20 @@
 package wrm;
 
-import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
```

Comment:
Sorry for superfluous changes in imports hereinafter -- IDE made automatic import optimisation on commit.